tests/tablecellproperties/tablecellpropertiesui.js

```diff
@@ -0,0 +1,344 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* globals document, Event */
+
+import ClassicTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/classictesteditor';
+import testUtils from '@ckeditor/ckeditor5-core/tests/_utils/utils';
+import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
+import { getData as getModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
+
+import Undo from '@ckeditor/ckeditor5-undo/src/undo';
+import Batch from '@ckeditor/ckeditor5-engine/src/model/batch';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import ContextualBalloon from '@ckeditor/ckeditor5-ui/src/panel/balloon/contextualballoon';
+
+import Table from '../../src/table';
+import TableCellPropertiesEditing from '../../src/tablecellproperties/tablecellpropertiesediting';
+import TableCellPropertiesUI from '../../src/tablecellproperties/tablecellpropertiesui';
+import TableCellPropertiesUIView from '../../src/tablecellproperties/ui/tablecellpropertiesview';
+
+describe( 'TableCellPropertiesUI', () => {
```

Comment:
I've wrapped the test in either `describe( 'table cell properties', () => {} )` or `describe( 'table properties', () => {} )` to group those tests. This is just cosmetic but it helped me to group those tests in a report:

![Selection_132](https://user-images.githubusercontent.com/247363/72888172-6107ed00-3d0d-11ea-93d7-63fcc996ded7.png)

vs current:
![Selection_133](https://user-images.githubusercontent.com/247363/72888228-7bda6180-3d0d-11ea-9519-2551d5f836a3.png)

and the same in CLI:
![Selection_134](https://user-images.githubusercontent.com/247363/72888304-9dd3e400-3d0d-11ea-8f61-23dd1a7775aa.png)

ps.: I'll hijack this comment for standup tension as current table tests are a mes :D